test/unit/pr_checker.test.js

```diff
@@ -80,12 +80,13 @@ describe('PRChecker', () => {
 
       const expectedLogs = {
         warn: [
+          ['Quux User(Quux)) approved in via LGTM in comments'],
```

Comment:
There is a redundant `)` here..

Reply:
Actually this redundant parens is in every test and is in fact in  the base template :D
I'll remove it everywhere.